library/psa_crypto_storage_its.c

```diff
@@ -26,6 +26,20 @@
 #endif
 
 #if defined(MBEDTLS_PSA_CRYPTO_STORAGE_ITS_C)
+/*
+ * When MBEDTLS_PSA_CRYPTO_SPM is defined, the code is being built for SPM
+ * (Secure Partition Manager) integration which separates the code into two
+ * parts: NSPE (Non-Secure Processing Environment) and SPE (Secure Processing
+ * Environment). When building for the SPE, an additional header file should be
+ * included.
+ */
+#if defined(MBEDTLS_PSA_CRYPTO_SPM)
+/*
+ * PSA_CRYPTO_SECURE means that this file is compiled for the SPE.
+ * Some headers will be affected by this flag.
```

Comment:
What is “some headers”? Since none of the code in this repository uses `PSA_CRYPTO_SECURE`, please explain what clause of the contract between Mbed Crypto and [some PSA component] this is about.

(I should have made this comment on the PR that originally introduced this wording, but it's still relevant.)